runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityConfig.java

```diff
@@ -234,12 +234,34 @@ default boolean highlightSepulchreNpcs()
 
 	@ConfigItem(
 		keyName = "sepulchreHighlightColor",
-		name = "Sepulchre Highlight",
+		name = "Projectile Color",
```

Comment:
Maybe add a "Hallowed Sepulchre" config section since "Projectile Color" on its own doesn't really mean anything in this plugin, and having four sepulchre configs is starting to get a bit busy at the bottom of this config.

Reply:
By adding the config section, it makes "Sepulchre" redundant in the config names, so those can probably be removed. Thoughts? 
<img width="237" alt="Screen Shot 2020-06-21 at 12 43 42 PM" src="https://user-images.githubusercontent.com/54762282/85230663-b715b500-b3bf-11ea-85cc-e2dc5d41f5d5.png">
<img width="234" alt="Screen Shot 2020-06-21 at 12 50 48 PM" src="https://user-images.githubusercontent.com/54762282/85230664-b9780f00-b3bf-11ea-80d8-3545c56d6629.png">

Reply:
I think that you have renamed this in the wrong commit.

Reply:
@Adam- Should hopefully be fixed now, thanks for pointing this out.